docs/guides/platform/disk-images/managing-the-kernel-on-a-linode/index.md

```diff
@@ -0,0 +1,210 @@
+{{</ note >}}
+
+1.  Navigate to your Linode in the [Cloud Manager](https://cloud.linode.com) and click the **Configuration** tab.
+
+1. Find your current *Configuration* and click the corresponding **Edit** link. This may be located within the **More Options Ellipses** dropdown menu.
+
+1.  Scroll to the *Boot Settings* section.
+
+1.  Review the **Kernel** dropdown menu selection.
+
+    - **Latest 64 bit**: Uses the latest 64-bit Linode kernel at the time the Linode boots/reboots. This was the default for most 64-bit distributions prior to August 2018.
+    - **Latest 32 bit**: Uses the latest 32-bit Linode kernel at the time the Linode boots/reboots. This was the default for most 32-bit distributions prior to August 2018.
+    - **Direct Disk**: Instead of a Linux Kernel, this uses the MBE (Master Boot Record) of the primary disk*.
+    - **GRUB 2**: Uses the upstream distribution-supplied kernel that's installed on the primary disk. In a custom kernel has been installed instead, that is used instead. **This is the most common option and has been the default for most new Linodes created after August 2018.**
+    - **GRUB (Legacy)**: Uses the upstream distribution-supplied kernel that's installed on the primary disk. This should only be used on older Linux distributions that have Grub (not Grub 2) installed, like CentOS 6.
+    - **Specific Linode Kernel**: Allows you to specify the exact Linode kernel to use. Since this kernel is maintained regardless of reboots or system updates, it may be preferred in some use cases.
+
+    *\*Primary disk: the disk assigned as the "Root Device" within the Linode's Configuration Profile.*
+
+1.  Make any desired changes and click the **Save Changes** button. Otherwise, if no changes were made, click **Cancel**.
+
+1.  If any changes were made, reboot your Linode for the new kernel to take affect.
+
+## Updating the Kernel
+
+Provided a newer kernel is available, you should be able to either manually (or automatically) update the kernel. The exact steps vary depending on the type of kernel and, for upstream kernels, the Linux distribution you are using.
+
+### Updating the Linode Kernel
+
+Follow these steps if the Linode is using a Linode kernel:
+
+-   **Latest 64-bit or 32-bit kernel**: Rebooting the Linode automatically updates the kernel used within your system.
+-   **Specific kernel version** (ex: `5.12.2-x86_64-linode144`): To update your kernel, follow the instructions within the [Viewing and Modifying the Kernel](#viewing-and-modifying-the-kernel) section. When selecting the kernel in the Linode's Configuration Profile, chose your desired kernel version (or select `Latest 64 bit`), save the changes, and reboot your Linode.
```

Comment:
The anchor is no longer working. It should be `#viewing-and-modifying-the-kernel-in-the-cloud-manager`